app/views/insured/families/_enrollment_refactored.html.erb

```diff
@@ -3,7 +3,7 @@
 <% if hbx_enrollment.is_coverage_waived? %>
   <%= render partial: 'insured/families/waived_coverage_widget', locals: { read_only: read_only || hbx_enrollment.can_make_changes? , hbx_enrollment: hbx_enrollment } %>
 <% else %>
-    <div class="hbx-enrollment-refactored-panel plan-tile panel panel-default module">
+    <div class="plan-tile panel panel-default module <%= "initially_hidden_enrollment hidden" if initially_hide_enrollment?(hbx_enrollment) %>">
```

Comment:
I think you removed by mistake the hbx-enrollment-refactored-panel class

Reply:
Good catch!